sagemaker-datawrangler/joined-dataflow/explore_data.ipynb

```diff
@@ -0,0 +1,281 @@
+{
```

Comment:
<p>Missing descriptions of how you are exploring the data and what you are exploring exactly. The audience may be confused at what you are trying to do here, especially for beginners</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>